frontend/src/book/store.js

```diff
@@ -132,17 +132,43 @@ export const setPageIndex = (pageIndex) => {
 
 export const movePage = (delta) => {
   setPageIndex(currentPage().index + delta);
+  if (delta < 0) {
+    removeStepsAnimations();
+  }
```

Comment:
When is this needed? I removed it and didn't see any animations when moving back.

Reply:
It is needed to cover a case when a user moves quickly back after submitting a step. Not very likely to happen, but in case it does, this will prevent the animation from playing. Here's how it looks when moving back fast without this: https://ibb.co/4W09VhR

Reply:
OK, I understand. Very meticulous! I think I'd prefer to remove this and keep the code simple. I'm fine with a harmless animation showing on extremely rare occasions.

Reply:
Sure thing. Removed.